android/modules/ui/src/java/ti/modules/titanium/ui/widget/listview/TiListView.java

```diff
@@ -615,6 +627,10 @@ public void processProperties(KrollDict d) {
 			footerView = inflater.inflate(headerFooterId, null);
 			footerView.findViewById(titleId).setVisibility(View.GONE);
 		}
+		
+		if (d.containsKeyAndNotNull(TiC.PROPERTY_CAN_SCROLL)) {
```

Comment:
@fmerzadyan Try to avoid rebasing of commits, it make it hard to see the history of a PR.